src/ui/public/inspector/ui/inspector_panel.test.js

```diff
@@ -59,7 +59,7 @@ describe('InspectorPanel', () => {
   });
 
   it('should render as expected', () => {
-    const component = mount(
+    const component = mountWithIntl(
```

Comment:
Why are you mounting the component with intl? The `InspectorPanel` doesn't seems to have the i18n injected with `injectI18n` method, so I don't see the need to mount it with this funciton

Reply:
The only thing you are currently translating in the panel is the title, and it's done by the `@kbn/i18n` package

Reply:
tried reverting jest tests to `mount` and they would fail with:
```
  ● InspectorPanel › should render as expected

    Error: Uncaught [Error: [React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry. Using default message as fallback.]
```

Reply:
Ok, I've understand: you are using a `FormattedMessage` on the tree of the panel and this requires the `IntlProvider`.
You can use the same set of tools for react also for formatting the title.
https://github.com/elastic/kibana/tree/master/packages/kbn-i18n#react